hubblestack/files/hubblestack_nova/stat_nova.py

```diff
@@ -101,14 +109,16 @@ def audit(data_list, tags, debug=False, **kwargs):
                 if not salt_ret:
                     if None in expected.values():
```

Comment:
I know this is not your fault, but I happened to notice this and I think that it's wrong.

I think this is what the author meant to do:

```
if not expected:
```

Otherwise a single argument with `None` will result in success. Can you evaluate this and make that change if you agree?

Reply:
I agree, I will make these changes as well.

Reply:
I agree, will make these changes as well.